src/network/networkd-network-gperf.gperf

```diff
@@ -130,6 +130,11 @@ BridgeFDB.VLANId,                       config_parse_fdb_vlan_id,
 BridgeVLAN.PVID,                        config_parse_brvlan_pvid,                       0,                             0
 BridgeVLAN.VLAN,                        config_parse_brvlan_vlan,                       0,                             0
 BridgeVLAN.EgressUntagged,              config_parse_brvlan_untagged,                   0,                             0
+Prefix.Prefix,                          config_parse_prefix,                            0,                             0
+Prefix.OnLink,                          config_parse_prefix_flags,                      0,                             0
+Prefix.AddressAutoconfiguration,        config_parse_prefix_flags,                      0,                             0
+Prefix.ValidLifetime,                   config_parse_prefix_uint32,                     0,                             0
+Prefix.PreferredLifetime,               config_parse_prefix_uint32,                     0,                             0
```

Comment:
Yes, the problem is that these values don't have a counterpart in IPv4 DHCP although they describe similar intentions. IPv6Prefix it is for now, but should someone come up with a uniform and descriptive naming before these patches are applied, it can be easily changed.